force-app/main/default/classes/SOQL.cls

```diff
@@ -142,16 +220,14 @@ public with sharing class SOQL {
      * @return A list of Contact LastName.
      */
     public static List<String> getContactNamesByAccount(Id accountId) {
-        // Create a list to hold the Contact LastNames
-
-        // Query the Contacts
-        // Loop through each Contact and add the LastName to the list
-        //for () {
-        //    
-        //}
-
-        // Return the list of Contact LastNames
-        return null; // Replace null with the variable you used to store the result
+        Contact[] myContacts = [SELECT Name from Contact where AccountId = :accountId];
+        List<String> myList= new List<String>();
+        if (myCOntacts.size()>0) {
+            for (Contact con : myContacts) {
+                myList.add(con.Name);
+            }
+        }
+            return myList;
```

Comment:
This is a good use case for a SOQL for loop.
Also, be careful with your code formatting and spacing.